src/averageAges.js

```diff
@@ -15,11 +15,12 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const man = people
+    .filter(person => person.sex === 'm')
+    .filter(person => !century || century === Math.ceil(person.died / 100))
```

Comment:
You do not need to filter twice. Just operate inside a method
```suggestion
    .filter(person => {
     const isMen = person.sex === 'm'
     
     if(...) {
       return ...
     }
    })
```